R/step_adjust_latency.R

```diff
@@ -84,8 +107,9 @@ step_adjust_latency <-
              "locf",
              "extend_lags"
            ),
+           epi_keys_checked = c("geo_value"),
            fixed_latency = NULL,
-           fixed_asof = NULL,
+           fixed_forecast_date = NULL,
            default = NA,
            skip = FALSE,
            columns = NULL,
```

Comment:
What does this do? Is it just populated by the tidy selection? You can inherit these from other `step_*` functions. (Same applies to `skip` and `id`)

Reply:
The default for `id` should be `rand_id("adjust_latency")`

Reply:
I was unsure which of these were necessary boilerplate for all steps, and which were args only the other step would need. Would gladly drop most of the generic steps if possible

Reply:
I've been basing it off the instructions here (though they may have changed since I last made one): 
https://www.tidymodels.org/learn/develop/recipes/

Reply:
oh you mean inherit the documentation probably?

Reply:
yes, I was wondering specifically if `columns` is necessary. The documentation makes it look like something set by the user, but it's actually ... not used at all?

Reply:
hmm, yeah `columns` is definitely a cargo-cult argument, I'll drop it. `...` are used to actually restrict the terms used to specific columns (though I don't have a test for that, going to add one to make sure it's working properly).

Reply:
That's ok. In many example `step_*()`, columns is an argument. I'm not honestly sure why. Take a look at 

```r
?recipes::step_lag
View(recipes::step_lag)
View(recipes:::step_lag_new)
View(recipes:::prep.step_lag)
```
The columns argument gets populated by `terms` at prep time. I see why they use it in `step_lag_new`, but I don't understand why it is kept as an argument to `step_lag`.

Long story short, they say to use it. But it needs to inherit the documentation. You can use
```r
#' @inheritParams recipes::step_lag
```